apps/web/src/components/Table/ResponsiveTable/RowSelection.tsx

```diff
@@ -246,31 +247,26 @@ const Actions = ({
                 >
                   &bull;
                 </span>
-                <span
-                  data-h2-position="base(relative)"
-                  data-h2-align-items="base(center) l-tablet(flex-start)"
-                  data-h2-padding="base(x.25)"
+                <DownloadCsv
+                  data-h2-font-weight="base(400)"
+                  {...download.selection.csv}
+                  {...actionButtonStyles}
                 >
-                  <DownloadCsv
-                    data-h2-font-weight="base(400)"
-                    {...download.selection.csv}
-                    {...actionButtonStyles}
-                  >
-                    {download.selection.label ||
-                      intl.formatMessage({
-                        defaultMessage: "Download CSV",
-                        id: "mxOuYK",
-                        description:
-                          "Text label for button to download a csv file of items in a table.",
-                      })}
-                  </DownloadCsv>
-                </span>
+                  {download.selection.label ||
+                    intl.formatMessage({
+                      defaultMessage: "Download CSV",
+                      id: "mxOuYK",
+                      description:
+                        "Text label for button to download a csv file of items in a table.",
+                    })}
+                </DownloadCsv>
               </span>
             )}
 
-            {print?.onPrint && (
+            {(print?.onPrint || print?.component) && (
```

Comment:
Does this work for you? I get error as `Property 'component' does not exist on type 'DatasetPrint`

Reply:
Hmm, it does seem to work for me and it does exist on that type:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/8f3366d60088aff1edbfb128e29e818fdd73a48b/apps/web/src/components/Table/ResponsiveTable/types.ts#L109-L113

And, it seems to function 🤔 
![Screenshot 2023-11-08 103639](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/77aa8ec6-6bf6-4a2b-8a0d-2f07c786aecb)